CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## v0.28.0 [2021-08-04]
+
+_What's new?_
+
+- New tables added
+  - [aws_emr_cluster_metric_isidle](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_emr_cluster_metric_isidle) ([#570](https://github.com/turbot/steampipe-plugin-aws/pull/570))
```

Comment:
Not a comment on the release note, but should we change this table name/file name to be `aws_emr_cluster_metric_is_idle` instead? Other metric tables have underscores in the metric names.